components/cli/pkg/kubernetes/set.go

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2019 WSO2 Inc. (http:www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http:www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package kubernetes
+
+import (
+	"os"
+	"os/exec"
+
+	"cellery.io/cellery/components/cli/pkg/constants"
+)
+
+func (kubeCli *CelleryKubeCli) SetNamespace(namespace string) error {
+	cmd := exec.Command(
+		constants.KubeCtl,
+		"config",
```

Comment:
Shall we move this function to config.go since the kubectl arg is "config".

Reply:
Fixed with https://github.com/wso2/cellery/pull/1011/commits/a61e1d8783d257e5e567cf2516a186a684125cc8.